filer/storage.py

```diff
@@ -53,4 +53,22 @@ def copy(self, src_name, dst_name):
 
 
 except ImportError:
-    pass
+    from storages.backends.s3boto3 import S3Boto3Storage
+
+    class PatchedS3BotoStorage(S3Boto3Storage):
```

Comment:
this class definition is the same as above, the only difference is the class it extends
perhaps we can simplify to (remove duplicate code):
```
try:
    from storages.backends.s3boto import S3BotoStorage
except ImportError:
    from storages.backends.s3boto3 import S3Boto3Storage as S3BotoStorage
 
class PatchedS3BotoStorage(S3BotoStorage)
```

Reply:
I'll make the changes. Thank you.